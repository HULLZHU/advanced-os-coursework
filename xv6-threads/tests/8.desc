test that ticket locks actually synchronizes
