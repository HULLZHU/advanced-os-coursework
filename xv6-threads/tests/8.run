cd src; ./../tester/run-xv6-command.exp CPUS=2 Makefile.test test_8 | grep XV6_TEST_OUTPUT; cd ..
