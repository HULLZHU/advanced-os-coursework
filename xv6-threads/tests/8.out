XV6_TEST_OUTPUT 0
